### PR TITLE
Issue #1033: Docs for for_each on import

### LINKED ISSUE #1033: Add support for `for_each` in the `import` block
### OpenTofu Version

```shell
1.6.0-dev
```

### Use Cases

The `import` block could benefit greatly from supporting `for_each`. 

Currently, importing resources that are using `for_each` is not possible / hard using the `import` block for them. Right now, to import those you'd need to use a script running `tofu import` commands

### Attempted Solutions

No possibility of importing `for_each` resources nicely using a single `import` block

### Proposal

By adding `for_each` for the `import` block, you could do the following:

```
import {
  for_each = local.bucket_ids
  id = each.value.id
  to = aws_s3_bucket.bucket[each.value.key]
}

resource "aws_s3_bucket" "bucket" {
  for_each = local.bucket_ids
  ...
}
```

### References

Related issue in the Terraform repository - https://github.com/hashicorp/terraform/issues/33624

### PR DESCRIPTION
**Pull request by ghost**
Resolves #1033

## Target Release

1.7.0

### CODE REVIEW

**Review by Yantrio — DISMISSED** [review]
I'm happy with this, I would recommend we wait for an approval from @Evi1Pumpkin too though.

Do we also have to mention tuples support?

File: website/docs/language/import/index.mdx
Comment (Evi1Pumpkin) [inline website/docs/language/import/index.mdx:138]: We also support `tuple` in the case of for_each. Using a tuple can be helpful when importing to a more complex address like `module.mod[each.value.moduleKey].test_object.a[each.value.resourceKey]`.

File: website/docs/language/import/index.mdx
Comment (Evi1Pumpkin) [inline website/docs/language/import/index.mdx:136]: Maybe we should mention that you cannot use the `Generate configuration` capability when we are importing resource using `for_each`

File: website/docs/language/import/index.mdx
Comment (Evi1Pumpkin) [inline website/docs/language/import/index.mdx:156]: This is an interesting example (I didn't even know it works), but shouldn't we use a more common way of calling for a for_each in our docs? Something more similar to [this](https://opentofu.org/docs/language/meta-arguments/for_each/#using-sets) for example.
I don't want people to be confused.

File: website/docs/language/import/index.mdx
Comment (Yantrio) [reply website/docs/language/import/index.mdx:136]: We should open an issue for that too. It would be a shame to not support that feature.

File: website/docs/language/import/index.mdx
Comment (Evi1Pumpkin) [reply website/docs/language/import/index.mdx:156]: Makes sense.
I used the `random_integer` resource while developing the resource. But I don't think it can be a very good example:
```
locals {
    numbers = {
      "first" = "0,1,0"
      "second" = "0,1,0"
    }
}

resource "random_integer" "a" {
  for_each = local.numbers
  min =	1
  max = 50000
}
```
Overall, I don't have a strong opinion about it, so I'm leaving that decision to you.

File: website/docs/language/import/index.mdx
Comment (ollevche) [inline website/docs/language/import/index.mdx:167]:
```suggestion
[Generating configuration](/docs/language/import/generating-configuration/) is currently not possible when using `for_each` on `import blocks.
```

**Review by ollevche — APPROVED** [review]

**Pull request merged**
